src/renderer/tvgRender.h

```diff
@@ -140,7 +140,7 @@ struct RenderStroke
     struct {
         float begin = 0.0f;
         float end = 1.0f;
-        bool individual = false;
+        bool simultaneous = true;
```

Comment:
@hermet I changed 'individual' to 'simultaneous' because imo the naming is very unfortunate. 'Individual' means that all paths within a given shape are treated as a whole, rather than individually, which is counterintuitive. 
It's easier to explain to the user that 'simultaneous' means that the trimming is done for all paths simultaneously if set to true, if false it is otherwise
but ofc I can restore the previous version if you don;t agree